stdlib/src/utils/stringref.mojo

```diff
@@ -136,6 +136,18 @@ struct StringRef(
 
         return DTypePointer[DType.uint8](ptr)
 
+    @always_inline
+    fn __init__(inout self, owned ptr: UnsafePointer[UInt8], length: Int):
```

Comment:
Q: Why is this needed? The `@value` decorator should generate a similar constructor, without `owned`.